fastestimator/op/numpyop/numpyop.py

```diff
@@ -264,7 +265,8 @@ def forward(self, data: List[np.ndarray], state: Dict[str, Any]) -> Optional[Fil
             return FilteredData(replacement=self.replacement)
         return None
 
-    def forward_batch(self, data: Union[Tensor, List[Tensor]], state: Dict[str, Any]) -> Optional[FilteredData]:
+    def forward_batch(self, data: Union[torch.Tensor, List[torch.Tensor]], state: Dict[str,
```

Comment:
We should keep this as data: Union[Tensor, List[Tensor]] because it could be np array